### ISSUE TITLE
K65 Lux Help!

### ISSUE THREAD

**Issue by SoupitySoup** [ccMSC/ckb#408]
Hey, i just got the new k65 lux and tried to get this to customize it. I downloaded the mac versions and did all the instructions, i downloaded the pkg, got Xcode, and xt5, but it still says no devices connected. so then i just downloaded the main one and it also says no devices connected after trying to reboot my computer and replug my keyboard but still nothing. I'm using a mac. and the main one, when i open it says "Warning: driver version mismatch) beta-v0.2.6). Please upgrade cab-daemon. If the problem persists, try rebooting.

**Comment by tatokis** [comment]
Install ckb from here https://github.com/ccMSC/ckb/archive/newdev.zip
There are instructions in the readme.

**Comment by SoupitySoup** [comment]
Nope, still says the same thing. Followed the instructions exactly as it says.

**Comment by mmsz111** [comment]
K70 Rapidfire
![screen shot 2017-01-28 at 10 50 00 am](https://cloud.githubusercontent.com/assets/25360189/22398171/5dd80ca2-e548-11e6-9c0b-c0135df0cb79.png)

help...plz I tried all branches and rebooted.

**Comment by mmsz111** [comment]
here... https://github.com/ccMSC/ckb/issues/570
